src/Components/NavItem/NavItem.js

```diff
@@ -1,42 +1,54 @@
-import styled from "styled-components/macro";
-import { NavLink } from "react-router-dom";
-
+import styled from 'styled-components/macro';
+import { NavLink } from 'react-router-dom';
 
 export default function NavItem() {
-
-    /*<NavLinkStyled to="/settings" activeClassName="active">Settings</NavLinkStyled> */
-
-    return <NavWrapper>
-        <NavLinkStyled exact to="/" activeClassName="active">Home</NavLinkStyled>
-        <NavLinkStyled to="/game" activeClassName="active">Game</NavLinkStyled>
-        <NavLinkStyled to="/highscore" activeClassName="active">High Score</NavLinkStyled>
+  /*<NavLinkStyled to="/settings" activeClassName="active">Settings</NavLinkStyled> */
```

Comment:
No comments in main! :)

Reply:
This is just there to save the code for later once I am implementing the settings page. :D